### ISSUE TITLE
Is there a way to clean *ycmd-server* buffer?

### ISSUE THREAD

**Issue by moyotar** [abingham/emacs-ycmd#459]

**Comment by abingham** [comment]
There's none right now, but I suppose we could add one. Is this just to make it easier to read? Or are you worried about it getting too big?

**Comment by moyotar** [comment]
Sorry for response late. For me, there are benefits for both of the results that you mentioned. I read it rarely if `ycmd` works finely.

**Comment by abingham** [comment]
In it's simplest formulation, this function would essentially be an alias for `erase-buffer`. Do you have an idea of when we should be clearing out the buffer? E.g. when it reaches a certain size, or after a certain amount of time? The hard part of this is not the function itself, but the policy around when it's called.

**Comment by moyotar** [comment]
I agree. How about to offer the two ways to make it alternative? I think it will satisfy most of us.

**Comment by abingham** [comment]
Sounds good to me.
